packages/web/app/pages/settings.tsx

```diff
@@ -6,6 +7,14 @@ import { Avatar, Button, Heading, Input, SubHeader, Tabs, Title } from '@/compon
 import { MeDocument } from '@/graphql';
 import { withSessionProtection } from '@/lib/supertokens/guard';
 
+gql(/* GraphQL */ `
+  fragment UpdateMeFragment on User {
+    id
+    fullName
+    displayName
+  }
+`);
```

Comment:
I added it here because I removed file that contain this fragment (he was nowhere used)